webrtc.html

```diff
@@ -1156,6 +1156,12 @@
                 might be configured to use local or private STUN or TURN
                 servers. This method allows an application to learn about these
                 servers and optionally use them.</p>
+                <p class="fingerprint">This list is likely to persisent and
```

Comment:
English: This list is likely to persistent -> This list is likely to be persistent